### PR TITLE
Added proposed Model methods

### PR DESCRIPTION
**Pull request by grapheo12**
@thealphadollar Please review. I could not understand how to implement the `isVerifiedEmail` without adding another column. Also comment on where to use the Loggers in `users.py` and `posts.py`.

### CODE REVIEW

File: app/main/models/users.py [hunk 1]
```diff
@@ -52,14 +54,82 @@ class User(db.Model):
...
+		db.session.commit()
+
+	def updateMetaData(self, **kwargs):
+		# TODO
+		pass
+	def getMetaData(self):
+		# TODO
+		pass
+
+	# TODO: Do something about isVerifiedEmail
```
Comment (thealphadollar) [inline app/main/models/users.py]: Add a database column that will store whether email is verified or not, and from here give `isVerified` and `setVerified` methods.

File: app/main/models/users.py [hunk 2]
```diff
@@ -52,14 +54,82 @@ class User(db.Model):
...
+	def getMetaData(self):
+		# TODO
+		pass
+
+	# TODO: Do something about isVerifiedEmail 
+
+	def setNewTag(self, tag):
+		self.tags.append(tag)
+		db.session.commit()
+	def setTagPriority(self, tag, priority):
```
Comment (thealphadollar) [inline app/main/models/users.py]: Leave a blank line after one definition ending.

File: app/main/models/imgLinks.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 """DB Model for imgLink table"""
 from . import db
-from app.main.config import imgdir
+from app.main.config import Config
```
Comment (thealphadollar) [inline app/main/models/imgLinks.py]: config is not used by importing Config class.

it is used as,

```
from flask import current_app

file = open(os.path.join(current_app.config['IMGDIR'], fname), 'wb)
```

File: app/main/models/imgLinks.py [hunk 4]
```diff
@@ -24,12 +24,12 @@ def __init__(self, imageBinary):
         [Needs to be changed]
         """
         fname = datetime.datetime.now() + ".png"
-        file = open(os.path.join(imgdir, fname), "wb")
+        file = open(os.path.join(Config.IMGDIR, fname), "wb")
```
Comment (thealphadollar) [inline app/main/models/imgLinks.py]: change here how you access config element

File: app/main/models/imgLinks.py [hunk 5]
```diff
@@ -24,12 +24,12 @@ def __init__(self, imageBinary):
...
         LOG.info("Writing new image to disk, %s", fname)
         file.write(imageBinary)
         file.close()
 
-        self.link = os.path.join(imgdir, fname)
+        self.link = os.path.join(Config.IMGDIR, fname)
```
Comment (thealphadollar) [inline app/main/models/imgLinks.py]: as above

File: app/main/models/imgLinks.py [hunk 6]
```diff
@@ -49,12 +49,12 @@ def update(self, newImgBin):
 
         self.id = id
         fname = datetime.datetime.now() + ".png"
-        file = open(os.path.join(imgdir, fname), "wb")
+        file = open(os.path.join(Config.IMGDIR, fname), "wb")
```
Comment (thealphadollar) [inline app/main/models/imgLinks.py]: as above

File: app/main/models/imgLinks.py [hunk 7]
```diff
@@ -49,12 +49,12 @@ def update(self, newImgBin):
...
         LOG.info("Writing new image to disk, %s", fname)
         file.write(newImgBin)
         file.close()
 
-        self.link = os.path.join(imgdir, fname)
+        self.link = os.path.join(Config.IMGDIR, fname)
```
Comment (thealphadollar) [inline app/main/models/imgLinks.py]: as above

File: app/main/models/payments.py [hunk 8]
```diff
@@ -24,12 +24,13 @@ def __init__(self, user, amount, api_response):
         self.amount = amount
         self.api_response = api_response
 
-        send_mail(user.email, "Thanks from Ambit", """
-We are very grateful to you. Blah Blah Blah.""")
+        async_send_mail(user.email, "Thanks from Ambit", """
```
Comment (thealphadollar) [inline app/main/models/payments.py]: This method has been added and it takes the current_app as an argument. Should be called as https://github.com/thealphadollar/TriNetra-Clone/blob/35f9de734052d0a0de232e13434ef7ca01a05663/app/main/controller/train_controller.py#L32

File: app/main/models/users.py [hunk 9]
```diff
@@ -52,6 +53,7 @@ class User(db.Model):
 	occupation = db.Column(db.String(255))
 	last_login = db.Column(db.DateTime)
 	creation_time = db.Column(db.DateTime)
+	is_verified = db.Column(db.Boolean)
```
Comment (thealphadollar) [inline app/main/models/users.py]: default value should be `False`

**Comment by thealphadollar** [comment]
@grapheo12 Updates?

**Comment by grapheo12** [comment]
@thealphadollar Sorry. I slacked off a bit for two days. I made all the changes you requested.

**Pull request merged**